src/sidebar/components/ThreadList.tsx

```diff
@@ -150,19 +150,6 @@ export default function ThreadList({ threads }: ThreadListProps) {
       [topLevelThreads, threadHeights, scrollPosition, scrollContainerHeight],
     );
 
-  const store = useSidebarStore();
-
-  // Get the `$tag` of the most recently created unsaved annotation.
-  const newAnnotationTag = (() => {
-    // If multiple unsaved annotations exist, assume that the last one in the
-    // list is the most recently created one.
-    const newAnnotations = store.unsavedAnnotations();
-    if (!newAnnotations.length) {
-      return null;
-    }
-    return newAnnotations[newAnnotations.length - 1].$tag;
-  })();
```

Comment:
I moved these down to be defined closer to the logic where they are actually used.